src/borg/archiver.py

```diff
@@ -3525,8 +3530,12 @@ def define_borg_mount(parser):
                                help='do not actually change any files')
         subparser.add_argument('--numeric-owner', dest='numeric_owner', action='store_true',
                                help='only obey numeric user and group identifiers')
+        # --nobsdflags is misleading and deprecated.
+        # use --noflags as a replacement.
         subparser.add_argument('--nobsdflags', dest='nobsdflags', action='store_true',
-                               help='do not extract/set bsdflags (e.g. NODUMP, IMMUTABLE)')
+                               help='DEPRECATED: do not extract/set bsdflags (e.g. NODUMP, IMMUTABLE)')
```

Comment:
instead of a source comment, rather point to --noflags from here, so it is visible on the UI.

Reply:
Yes. I saw the --noatime comments few lines above and added some comments, too.